### DISCUSSION TITLE
Support request: Can't anymore install or update cw

### DISCUSSION THREAD

**Discussion by electroheadfx** [aws/codewhisperer-command-line-discussions#160] [Support ticket]
### Checks

- [X] I ran `q doctor` in the affected command line session and it didn't resolve my issue
- [X] I ran `q restart` and it didn't resolve my issue
- [X] I searched [the discussions in this repo](https://github.com/aws/q-command-line-discussions/discussions?q=) and there are no duplicates of my issue

### Expected behaviour

Update or install CodeWhisperer. It worked before on my machine.

### Actual behaviour

I try to run cw doctor, it update and it fails, 2 times and it quit.
![Capture d’écran 2024-05-02 à 19 01 21](https://github.com/aws/q-command-line-discussions/assets/1211149/604ed492-ab79-4f6a-afbd-98100f4a57d3)

I tried to un-install and re-install cw from amazon, it force me to update and fail !

Iam lost, there is Q and CW software, is there a clean install ???

![Capture d’écran 2024-05-02 à 19 11 11](https://github.com/aws/q-command-line-discussions/assets/1211149/965f3fb7-93d7-4442-bc77-a4bc4ca975a5)

### Environment

```yaml
MacOS 14.4.1 / M3 Max
```

**Comment by electroheadfx** [comment]
I unstalled Q then I installed codeWispered from image, it worked, but it seem cw was replaced by Q ?

**Comment by grant0417** [comment]
Sorry for the issue, we have resolved the issue with Amazon Q 1.0.1 which you can download here: https://docs.aws.amazon.com/amazonq/latest/qdeveloper-ug/command-line-getting-started-installing.html

I would recommend removing the old CodeWhisperer.app or Q.app before installing.
